### PR TITLE
Add count of files converted

### PR DESCRIPTION
**Pull request by traciklopez**
Hoping to add a new datadog report that measures the number of files converted from JS to Typescript for a given author. We're(I'm) trying to gamify the migration process with a "leader board". First time doing a datadog report. Feedback is welcomed and encouraged. 

Couple things to note - added Jest so I could write a very simple test. Also am only checking for the exact name of the file. Say you migrate ManageFlagTargeting but move it into its own directory. That will not count as a point.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -4,7 +4,10 @@ inputs:
   source-path: 
     description: 'The source directory'
     required: true
-  datadog-metric:
+  datadog-typescript-progress-metric:
+    description: 'The Datadog metric name to submit to'
```
Comment (atrakh) [inline action.yml:8]: Could we distinguish the descriptions for the 2 metric names?

File: src/utils/helperMethods.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export function findFileCountOfJSConversionsToTS(filesAdded: string[], filesRemoved: string[]) {
+  let count = 0;
+  filesAdded.forEach((d) => {
+    const splitFileName = d.split('.');
+    filesRemoved.forEach((e) => {
```
Comment (atrakh) [inline src/utils/helperMethods.ts:5]: Might be able to use a `filesRemoved.find(<find matching file>)` here to avoid a nested loop

**Comment by traciklopez** [comment]
Here's the gonfalon test pr for the github action

**Review by apucacao — APPROVED** [review]

**Pull request merged**